proto/eventrecorder/events/v2/events.proto

```
@@ -74,13 +68,13 @@ message AlertCreatedEvent {
 
 // AlertResolvedEvent is emitted when a resolved alert leaves its group.
 message AlertResolvedEvent {
-  GroupedAlert alert = 1;
+  Alert alert = 1;
```

Buf (1.72.0)

[error] Field "1" with name "alert" on message "AlertResolvedEvent" changed type from "events.v2.GroupedAlert" to "events.v2.Alert".

(FIELD_WIRE_JSON_COMPATIBLE_TYPE)